source/numerics.tex

```diff
@@ -10085,9 +10085,9 @@
 and for positive infinity.
 
 \rSec2[sf.cmath.Lnm]{Associated Laguerre polynomials}%
-\index{assoc_laguerre@\tcode{assoc_laguerre}}%
-\index{Laguerre polynomials $\mathsf{L}_n^m$}%
-\index{L nm@$\mathsf{L}_n^m$ (associated Laguerre polynomials)}%
+\indexlibrary{\idxcode{assoc_laguerre}}%
```

Comment:
Would you mind also adding index entries for the `l`- and `f`-suffixed forms here?